app/assets/javascripts/hidden_until_select.js.coffee

```diff
@@ -0,0 +1,16 @@
+class HiddenUntilSelect
+  constructor: ($root, $hidden_el) ->
+    @$hidden_el = $hidden_el
```

Comment:
Two very minor things:

* CoffeeScript methods can take a property name prefixed with `@`, which automatically does assignment. These two lines could be collapsed to

    ```javascript
    constructor: ($root, @$hidden_el) ->
    ```

* In JS, the convention is to use camelCase names for variables (`$hiddenEl` vs. `$hidden_el`)